linode_api4/objects/linode.py

```diff
@@ -348,6 +348,13 @@ def reboot(self):
             return False
         return True
 
+    def resize(self, new_type):
+        resp = self._client.post("{}/resize".format(Instance.api_endpoint), model=self, data={"type": new_type})
```

Comment:
Can you make `new_type` work like [`ltype` does here](https://github.com/linode/linode_api4-python/blob/master/linode_api4/linode_client.py#L251)?  This library generally accepts _either_ an object or an ID for everything, so both of these should work:

```python
linode.resize('g6-standard-1')

linode.resize(client.linode.types().first())
```